PEP 758: Force parentheses when using 'as' (#4073)
python/peps: peps/pep-0758.rst

    except* ExceptionA, ExceptionB, ExceptionC:
        ...

When using the ``as`` clause to capture the exception instance paretheses must

> be used as before. Some users have expressed that they would find confusing not

Reviewer: ```suggestion
be used as before. Some users have expressed that they would find it confusing not
```

   This was rejected due to the potential for confusion and to maintain a clear
   distinction between the two styles.

Deferred Ideas
==============

1. Allowing unparenthesized expressions when the ``as`` keyword is used. The reason

>    we have decided to deferr this particular form is that given that there isn't clear

Reviewer: ```suggestion
   we have decided to defer this particular form given that there isn't clear
```

   consensus either way and there are reasonable arguments for both positions, the safest
   approach is keeping the parentheses requirement since it can be removed later if users

>    find the disconnect too accute, while taking it out and users deciding it was a bad idea

Reviewer: ```suggestion
   find the disconnect too acute, while taking it out and users deciding it was a bad idea
```
